doc/config_rsmexplain.rst.inc

```diff
@@ -0,0 +1,66 @@
+
+custom_sections *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A list of custom, user-defined sections to be included into the final report. These are IPython notebooks (``.ipynb`` files) created by the user.  The list must contains paths to the notebook files, either absolute or relative to the configuration file. All custom notebooks have access to some :ref:`pre-defined variables <custom_notebooks>`.
+
+description *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~
+A brief description of the ``rsmexplain`` experiment that will be shown at the top of the report. The description can contain spaces and punctuation.
+
+num_features_to_display *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Number of top features that should be displayed in ``rsmexplain`` plots. Defaults to ``15``.
+
+id_column *(Optional)*
+~~~~~~~~~
+The name of the column containing the response IDs. Defaults to ``spkitemid``, i.e., if this is not specified, ``rsmexplain`` will look for a column called ``spkitemid`` in both background_data and explainable_data files. Note: id_column must be same in these two files.
+
+sample_range *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~
+If we want to explain a specific range of responses from the ``explainable_data``, we can specify that range here. Note that the range is specified in terms of the *location* of the responses in the ``explainable_data`` file and that the locations are zero-indexed. So, for example, to explain only the first 50 responses in the file, we should set a value of ``"0-49"`` for this option. Defaults to ``None``.
+
+sample_size *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~
+If we want to explain a random sample of the responses in explainable_data, we can specify the size of that random sample here. For example, to explain a random sample of 10 responses, we would set this to 10. Defaults to None.
+
+.. note ::
+
+    Only one of ``sample_range`` or ``sample_size`` must be specified. If neither is specified, explanations will be generated for the entire set of responses in ``explainable_data`` which could be very slow, depending on its size.
+
+show_auto_cohorts *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If this option is set to ``true``, auto cohort bar plots will be displayed. These plots can be useful to detect interaction effects between cohorts and features. If a cohort shows a high feature value, then there may be an interaction between that cohort and the feature. Defaults to ``false``.
+
+standardize_features *(Optional)*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If this option is set to ``false``, the feature values for the responses in ``background_data`` and ``explainable_data`` will *not* be standardized using the mean and standard deviation parameters from the rsmtool experiment from ``<experiment_dir>/output/<experiment_id>_feature.csv``. This should only be set to ``false`` if the same option was also set to ``false`` when running the original rsmtool experiment. Defaults to ``true``.
```

Comment:
if it was set to false in the original experiment, it *must* be set to false here?
can we tell that this was the case from the original experiment output?

Reply:
We can if we have the full output of the original experiment including the saved configuration file. I guess we should try to see if we have that available and try to read it, if it is.

Reply:
I think a requirement to run this following an rsmtool experiemnt with all its input and output data in the expected format is reasonable, we already assume part of it.

Reply:
Fair point. I created an issue #602 . @damien2012eng  do you feel comfortable taking a stab at this?